explainer-server/app/db/ExplainerDB.scala

```diff
@@ -42,6 +42,8 @@ class ExplainerDB @Inject() (config: Config) extends ExplainerAtomImplicits {
         body = conversionFunction(explainer.tdata.body))))
   }
 
+  def stringConvertIterator(i: Iterator[Atom]) = i.toSeq.map(emptyStringConversion(_, emptyStringMarkerToEmptyString))
```

Comment:
I could do, but I get an Iterator out of the database, so at some point I'll need to do a .toSeq. Putting it here rather than in the functions that call it at least means its only in one place.